config/jobs/kubernetes/enhancements/enhancements-presubmit.yaml

```diff
@@ -10,3 +10,13 @@ presubmits:
               - sh
               - "-c"
               - "export PATH=$PATH:$GOPATH/bin && ./hack/verify.sh"
+    - name: pull-enhancements-test
+      always_run: true
+      decorate: true
+      spec:
+        containers:
+          - image: golang:1.17
```

Comment:
This ends up running one of the same tests that verify does.  Why not bump the verify job to the same golang image? To reduce blast radius.

Ideally the verify job would be `make verify` but there are duplicate targets and not all of them pass.  So since fixing that requires kubernetes/enhancement changes, hold off on bumping the verify image.

Reply:
Heh. I was just about to ask this :)